docs/src/main/asciidoc/se/webclient.adoc

```diff
@@ -260,30 +250,26 @@ include::{rootdir}/config/io_helidon_webclient_http2_Http2ClientProtocolConfig.a
 
 [source,java]
 ----
-Config config = Config.create();
-WebClient client = WebClient.builder()
-        .baseUri("http://localhost")
-        .config(config.get("client"))
-        .build();
+include::{sourcedir}/WebClientSnippets.java[tag=create-with-config, indent=0]
 ----
 
 === Example of a WebClient YAML Configuration
 
-[source, java]
+[source, yaml]
 ----
 client:
   connect-timeout-millis: 2000
   read-timeout-millis: 2000
-  follow-redirects: true <1>
+  follow-redirects: true # <1>
   max-redirects: 5
-  cookie-manager: <2>
+  cookie-manager: # <2>
     automatic-store-enabled: true
     default-cookies:
       flavor3: strawberry
       flavor4: raspberry
   default-headers: <3>
-    Accept: '"application/json","text/plain"'
-  services: <4>
+    Accept:| "application/json", "text/plain"
```

Comment:
It's correct but not needed. I did that because IntelliJ reported an error with the snippet.
It turns out that the `<3>` above needs to be after a line comment (`# <3>`) ; I will fix the annotation and revert the change for `Accept`.